src/chttpd/src/chttpd.erl

```diff
@@ -1036,14 +1036,16 @@ error_headers(#httpd{mochi_req=MochiReq}=Req, 401=Code, ErrorStr, ReasonStr) ->
             % redirect to the session page.
             case ErrorStr of
             <<"unauthorized">> ->
-                case config:get("couch_httpd_auth", "authentication_redirect", undefined) of
+                case chttpd_util:get_chttpd_auth_config(
+                    "authentication_redirect", "/_utils/session.html") of
```

Comment:
I am not sure about this change. Previously if "authentication_redirect" was not configured we returned just an error code. With this change we would always use redirect. I would like to hear what others think.

Reply:
In the default.ini file, `authentication_redirect = / _utils/session.html`.
I think config:get("couch_httpd_auth", "authentication_redirect", undefined) always returns `/_utils/session.html`, that is why I changed it here.

Except for "authentication_redirect", "iterations" have different values, too.
    default.ini: `iterations = 10`
    previous code: `config:get("couch_httpd_auth", "iterations", "1000")`

Thank you for review, @iilyak

Reply:
Some people may have different content in default.ini. For them it would be a big surprise why after update things started to work differently.

Reply:
If they set a new value in "default.ini", they will get the value that has been set. (personal settings)
However, if they delete the setting, they will get the `Default` value (10 or "/_utils/session.html").

Reply:
Exactly. Previously if they didn't have settings in the default.ini they wouldn't be redirected at all. Now they will be.

Reply:
default.ini is shipped and installed with CouchDB packages and is overwritten by package installs.
In the previous "default.ini", `authentication_redirect = /_utils/session.html`, `iterations = 10`
https://github.com/apache/couchdb/pull/3577/files#diff-38acf80292ea264190d7a08646c46d571d7ddcb37a749a58fb79c009030dd115L284-L289

@wohali Do you have a better idea, how to handle `authentication_redirect` and `iterations`?